authui/src/authflowv2/components/settings-user-profile-pic.css

```diff
@@ -0,0 +1,18 @@
+@layer component {
+  .settings-user-profile-pic__image-container {
+    @apply relative;
+    @apply h-22 w-22 tablet:h-50 tablet:w-50;
+  }
+
+  .settings-user-profile-pic__edit-button {
+    @apply inline-flex items-center justify-center;
+    @apply w-9 h-9;
+    @apply absolute bottom-0 -right-2 tablet:right-3.5;
+    @apply rounded-full bg-white;
+    box-shadow: 0 4px 10px 0 #0000001a;
```

Comment:
Is `#0000001a` from Figma? Does it has a name in the Figma file?

Reply:
No name for this

Reply:
Add a token to tailwind.config.js `shadow-settings-user-profile-pic-edit-button`